llpc/CMakeLists.txt

```diff
@@ -43,12 +43,11 @@ endif()
 
 ### Set Options and build LLVM #########################################################################################
 if(ICD_BUILD_LLPC)
-    # Set LLVM options and build LLVM
-    # Add LGC as an LLVM external project, but only if its CMakeLists.txt exists.
-    set(LLVM_EXTERNAL_PROJECTS llvm-dialects lgc LgcRt LgcCps)
-    set(LLVM_EXTERNAL_LGCCPS_SOURCE_DIR ${PROJECT_SOURCE_DIR}/../shared/lgccps)
-    set(LLVM_EXTERNAL_LLVM_DIALECTS_SOURCE_DIR ${PROJECT_SOURCE_DIR}/../imported/llvm-dialects)
-    set(LLVM_EXTERNAL_LGC_SOURCE_DIR ${PROJECT_SOURCE_DIR}/../lgc)
+    # Add LGC and its dependencies as LLVM external projects.
+    include("${CMAKE_CURRENT_SOURCE_DIR}/../cmake/lgc.cmake")
```

Comment:
Is the CMAKE_CURRENT_SOURCE_DIR really necessary?

Reply:
No. :-)